winui/Rating/Precision.md

```diff
@@ -1,6 +1,6 @@
 ---
 layout: post
-title: Precision with WinUI Rating control | Syncfusion
+title: Precision in WinUI Rating control | Syncfusion
 description: Learn here all about the precision feature in the Rating control to set the accuracy level such as full, half, and exact.
```

Comment:
description: Learn all about the precision feature in the Rating control to set the accuracy level such as full, half, and exact here.

Reply:
Changed the content as mentioned above.